src/test/java/com/wikia/webdriver/testcases/discussions/TrackingOptIn.java

```diff
@@ -79,57 +79,57 @@ public void anonInEUOnRejectShouldGetAnonymizedGoogleTracking(List<String> urlPa
     public void testModalVisibilityForAnonOnMobile() {
-        TrackingOptInModal trackingModal = setGeoCookieToGermanyAndNavigate();
+        TrackingOptInPage trackingModal = setGeoCookieToGermanyAndNavigate();
         Assertion.assertTrue(trackingModal.isVisible());
     }
 
     @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
     @Test(groups = {"discussions-tracking-opt-in-mobile"})
     @Execute(asUser = User.USER)
     public void loggedInEUShouldNotGetModalIfOptedInOnMobile() {
-        TrackingOptInModal trackingModal = setGeoCookieToGermanyAndNavigate();
+        TrackingOptInPage trackingModal = setGeoCookieToGermanyAndNavigate();
         Assertion.assertFalse(trackingModal.isVisible());
     }
 
     @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
     @Test(groups = {"discussions-tracking-opt-in-mobile"})
     @Execute(asUser = User.ANONYMOUS)
     public void AnonInEUShouldNotGetModalIfOptedInOnMobile() {
-        TrackingOptInModal trackingModal = setGeoCookieToGermanyAndNavigate();
+        TrackingOptInPage trackingModal = setGeoCookieToGermanyAndNavigate();
         Assertion.assertFalse(trackingModal.isVisible());
     }
 
     @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
     @Test(groups = {"discussions-tracking-opt-in-mobile"})
     @Execute(asUser = User.USER, trackingOptIn = false, trackingOptOut = true)
     public void loggedInEUShouldNotGetModalIfOptedOutOnMobile() {
-        TrackingOptInModal trackingModal = setGeoCookieToGermanyAndNavigate();
+        TrackingOptInPage trackingModal = setGeoCookieToGermanyAndNavigate();
         Assertion.assertFalse(trackingModal.isVisible());
     }
 
     @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
     @Test(groups = {"discussions-tracking-opt-in-mobile"})
     @Execute(asUser = User.ANONYMOUS, trackingOptIn = false, trackingOptOut = true)
     public void AnonInEUShouldNotGetModalIfOptedOutOnMobile() {
-        TrackingOptInModal trackingModal = setGeoCookieToGermanyAndNavigate();
+        TrackingOptInPage trackingModal = setGeoCookieToGermanyAndNavigate();
         Assertion.assertFalse(trackingModal.isVisible());
     }
 
     /*
         HELPERS
     */
 
-    private TrackingOptInModal setGeoCookieToGermanyAndNavigate() {
-        TrackingOptInModal trackingModal = new TrackingOptInModal();
+    private TrackingOptInPage setGeoCookieToGermanyAndNavigate() {
+        TrackingOptInPage trackingModal = new TrackingOptInPage();
         trackingModal.setGeoCookie(driver, "EU", "DE");
         new PostsListPage().open();
         trackingModal.logTrackingCookieValue();
 
         return trackingModal;
     }
 
-    private TrackingOptInModal setGeoCookieAndNavigate(String continent, String country) {
-        TrackingOptInModal trackingModal = new TrackingOptInModal();
+    private TrackingOptInPage setGeoCookieAndNavigate(String continent, String country) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "setGeoCookieAndNavigate" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

Reply:
Is Sonar right? If yes, remove it ;)